generator/ServiceClientGeneratorLib/DefaultConfiguration/DefaultConfigurationModel.cs

```diff
@@ -91,18 +91,11 @@ public enum RequestRetryMode
     }
 
     /// <summary>
-    /// Sts Regional Endpoints Value determines whether or not
-    /// to send the sts request to the regional endpoint or to
-    /// the global sts endpoint
+    /// StsRegionalEndpointsValue determines the service endpoint used to talk to STS.
     /// </summary>
     /// <remarks>This is a copy of Amazon.Runtime.StsRegionalEndpointsValue</remarks>
     public enum StsRegionalEndpointsValue
```

Comment:
Do we even need this enum anymore or can we just delete it. I think we can just delete it because the only reason we had it before was to support the legacy use case. Without the legacy endpoint resolution STS works like any other service and shouldn't need this enum.

Reply:
I can delete the enums; I don't expect any new values to be added, but then we'd ignore the `AWS_STS_REGIONAL_ENDPOINTS` / `sts_regional_endpoints` options and the value from the default configuration file: https://github.com/aws/aws-sdk-net/blob/main/sdk/src/Core/sdk-default-configuration.json#L5

Do you have any concerns with that?

Reply:
I found the internal design for `STS Regionalized Endpoints`, seems safe to remove it:
> SDK major version bumps should ignore the STS Regional Endpoints option, and only resolve STS endpoints to the partition's STS regional endpoints. A major version bump SDK may choose to keep the STS Regional Endpoints option, but the option's default value must be regional.

Removed both enums (from Core and the generator) in: https://github.com/aws/aws-sdk-net/pull/3236/commits/d17f17508101fa70e4ebcabecdc5286e446205aa

This also changed the generated files, included them in a separate commit: https://github.com/aws/aws-sdk-net/pull/3236/commits/cfdf9f98777e79f443fde4823e7b77e009ed2471